### PR TITLE
Add CSV output to the lwaftr loadtest command

### PR DESCRIPTION
**Pull request by teknico**
The `lwaftr loadtest` command currently only prints human-readable data to stdout: add data output to a CSV file in machine-readable format.

### CODE REVIEW

File: src/program/lwaftr/transient/transient.lua [hunk 1]
```diff
@@ -64,7 +64,7 @@ function parse_args(args)
    function handlers.h() show_usage(0) end
    args = lib.dogetopt(args, handlers, "hb:s:D:p:",
                        { bitrate="b", step="s", duration="D", period="p",
-                         ["bench-file"]=0, help="h" })
+                         ["bench-file"]=1, help="h" })
```
Comment (teknico) [inline src/program/lwaftr/transient/transient.lua:67]: Unrelated fix: different command, same option.

**Comment by kbara** [comment]
LGTM

**Comment by teknico** [comment]
That was quick, thank you. :-)
